<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H3rHelperClass</class>
 <widget class="QMainWindow" name="H3rHelperClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H3rHelper</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vertical_main">
     <item>
      <widget class="QGraphicsView" name="logo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group">
       <property name="title">
        <string>认证选项</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontal_sno">
         <item>
          <widget class="QLabel" name="label_sno">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>学号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_sno"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontal_code">
         <item>
          <widget class="QLabel" name="label_code">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_code">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>301</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontal_service">
         <item>
          <widget class="QLabel" name="label_service">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>目标服务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combo_service">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>...</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[预留] NWPU-LIB</string>
            </property>
            <property name="icon">
             <iconset resource="H3rHelper.qrc">
              <normaloff>:/H3rHelper/Resources/srun.png</normaloff>:/H3rHelper/Resources/srun.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NWPU-WLAN</string>
            </property>
            <property name="icon">
             <iconset resource="H3rHelper.qrc">
              <normaloff>:/H3rHelper/Resources/srun.png</normaloff>:/H3rHelper/Resources/srun.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[弃用] NWPU-LIB</string>
            </property>
            <property name="icon">
             <iconset resource="H3rHelper.qrc">
              <normaloff>:/H3rHelper/Resources/h3r.ico</normaloff>:/H3rHelper/Resources/h3r.ico</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[弃用] NWPU-WLAN (137)</string>
            </property>
            <property name="icon">
             <iconset resource="H3rHelper.qrc">
              <normaloff>:/H3rHelper/Resources/h3r.ico</normaloff>:/H3rHelper/Resources/h3r.ico</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>[弃用] NWPU-WLAN (138)</string>
            </property>
            <property name="icon">
             <iconset resource="H3rHelper.qrc">
              <normaloff>:/H3rHelper/Resources/h3r.ico</normaloff>:/H3rHelper/Resources/h3r.ico</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_detect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>识别</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="text_log">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontal_btn1">
         <item>
          <widget class="QPushButton" name="button_self">
           <property name="text">
            <string>自助服务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_info">
           <property name="text">
            <string>状态查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_toggle">
           <property name="text">
            <string>切换</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontal_btn2">
         <item>
          <widget class="QPushButton" name="button_about">
           <property name="text">
            <string>关于</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_clear">
           <property name="text">
            <string>清空日志</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_hide">
           <property name="text">
            <string>隐藏</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>301</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontal_action">
         <item>
          <widget class="QWidget" name="widget_action" native="true">
           <widget class="QWidget" name="horizontalLayoutWidget_7">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>301</width>
              <height>21</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_action">
               <property name="maximumSize">
                <size>
                 <width>47</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>认证操作</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_login">
               <property name="text">
                <string>上线</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_logout">
               <property name="text">
                <string>下线</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="H3rHelper.qrc"/>
 </resources>
 <connections/>
</ui>
